django_app/redbox_app/redbox_core/admin.py

```diff
@@ -86,21 +87,34 @@ class ChatMessageInline(admin.StackedInline):
 
 
 class ChatHistoryAdmin(admin.ModelAdmin):
-    def export_as_csv(self, request, queryset):  # noqa:ARG002
-        history_field_names = [field.name for field in models.ChatHistory._meta.fields]  # noqa:SLF001
-        message_field_names = [field.name for field in models.ChatMessage._meta.fields]  # noqa:SLF001
+    def export_as_csv(self, request, queryset: QuerySet):  # noqa:ARG002
+        history_field_names: list[str] = [field.name for field in models.ChatHistory._meta.fields]  # noqa:SLF001
+        message_field_names: list[str] = [field.name for field in models.ChatMessage._meta.fields]  # noqa:SLF001
+        rating_field_names: list[str] = [field.name for field in models.ChatMessageRating._meta.fields]  # noqa:SLF001
 
         response = HttpResponse(content_type="text/csv")
         response["Content-Disposition"] = "attachment; filename=chathistory.csv"
         writer = csv.writer(response)
 
-        writer.writerow(history_field_names + message_field_names)
+        writer.writerow(
+            ["history_" + n for n in history_field_names]
+            + ["message_" + n for n in message_field_names]
+            + ["rating_" + n for n in rating_field_names]
+            + ["rating_chips"]
+        )
+        chat_history: models.ChatHistory
+        chat_message: models.ChatMessage
+        chat_massage_rating: models.ChatMessageRating
```

Comment:
think this is a typo?

Reply:
I'm going to pretend it's a deliberate Monty Python reference - https://www.youtube.com/watch?v=P75lkpL7aGk - but I will fix.